observability/readme.md

```diff
@@ -12,7 +12,8 @@ Building observable systems enables development teams at CSE to measure how well
 ## Sections
 
 - [Pillars of Observability](#pillars-of-observability)
-- [Observability in Machine Learning](ml-observability.md)
+- [Observability in Machine Learning](observability/ml-observability.md)
```

Comment:
Maybe I'm misunderstanding so correct me if I'm wrong - this link wasn't broken, but with the new change it gives me a 404.